site/help/faq.md

```diff
@@ -110,17 +112,19 @@ Two websites have a description of the copyright process, and the various checks
 
 ### What other things has Project Gutenberg published in the past?
 
-We have published some music files, in MIDI and MUS formats. We have published the Human Genome. We have published pictures of the prehistoric cave painting from the south of France. We have published some video files and some audio files, including tracks by award-winning singer/songwriter Janis Ian and readings from public domain books.
+We have published music files, in MIDI and MUS formats. We have published the Human Genome. We have published pictures of the prehistoric cave painting from the south of France. We have published video files and audio files, including tracks by award-winning singer/songwriter Janis Ian and readings from public domain books.
+
+Today the primary focus is to distribute literary works that were previously published and are now in the public domain. The [collection development](/policy/collection_development.html) policy has details.
 
-Now, however, we only publish literary works that were previously published. The [collection development](/policy/collection_development.html) policy has details.
+In 2011 we began the Project Gutenberg Self Publishing Portal to focus on distribution of new works. Since then the self-publishing portal has released over 20,000 titles. [self.gutenberg.org](http://self.gutenberg.org/)
 
 ### How does Project Gutenberg choose books to publish?
 
 Project Gutenberg, as such, does not choose books to publish. There is no central list of works that volunteers are asked to work on. Individual volunteers choose and produce eBooks from printed books according to their own tastes and values, and the availability of a suitable printed book for digitization.
 
```

Comment:
Mention distributed proofreaders?

Reply:
That wasn't originally in this part of the FAQ so it didn't cross my mind to expand it in this PR. I will add that to my notes for later revisions, after this update.

Reply:
ok, I'll put something in.